### PR TITLE
Update ServiceBus SDK from t1 to t2

### PR DESCRIPTION
**Pull request by v-yilinhu**
1. Update ServiceBus SDK from t1 to t2

2. Update files:

    - ./provision-team/svcbusclient/Program.cs
    - ./provision-team/svcbusclient/svcbusclient.csproj

@scottaddie, @AlexGhiondea, @lilyjma, @ramya-rao-a and @JoshLove-msft for notification.

### CODE REVIEW

File: provision-team/svcbusclient/Program.cs [hunk 1]
```diff
@@ -47,17 +47,18 @@ static void Main(string[] args)
...
             await SendMessagesAsync(recipientEmail, MessageBody);
 
-            await queueClient.CloseAsync();
+            await sender.CloseAsync();
         }
 
         static async Task SendMessagesAsync(string recipientEmail, string messageBody)
         {
-            Random rnd = new Random();
+            //Random rnd = new Random();
```
Comment (JoshLove-msft) [inline provision-team/svcbusclient/Program.cs:61]: If this isn't used, please remove.

File: provision-team/svcbusclient/Program.cs [hunk 2]
```diff
@@ -68,17 +69,12 @@ static async Task SendMessagesAsync(string recipientEmail, string messageBody)
...
-                    var serializer = DataContractBinarySerializer<Dictionary<string, string>>.Instance;
-                    byte[] data;
+                    string data = JsonConvert.SerializeObject(message);
 
-                    using (MemoryStream stream = new MemoryStream())
-                    {
-                        serializer.WriteObject(stream, message);
-                        data = stream.ToArray();
-                    }
+                    ServiceBusMessage sendMessage = new ServiceBusMessage(data);
```
Comment (JoshLove-msft) [inline provision-team/svcbusclient/Program.cs:74]: We should use the built-in integration with System.Text.Json by leveraging the BinaryData type:
new ServiceBusMessage(new BinaryData(message)

Comment (JoshLove-msft) [inline provision-team/svcbusclient/Program.cs:74]: Rename `message` dictionary to `data`

File: provision-team/svcbusclient/svcbusclient.csproj [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="3.1.0" />
+    <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.5.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```
Comment (JoshLove-msft) [inline provision-team/svcbusclient/svcbusclient.csproj:10]: Remove and just use BinaryData.

File: provision-team/svcbusclient/Program.cs [hunk 4]
```diff
@@ -68,17 +69,12 @@ static async Task SendMessagesAsync(string recipientEmail, string messageBody)
                         { "Message", messageBody }
                     };
 
-                    var serializer = DataContractBinarySerializer<Dictionary<string, string>>.Instance;
-                    byte[] data;
+                    string data = JsonConvert.SerializeObject(message);
```
Comment (JoshLove-msft) [inline provision-team/svcbusclient/Program.cs:72]: Not needed

Comment (v-yilinhu) [reply provision-team/svcbusclient/Program.cs]: @JoshLove-msft, I have updated, please review.

**Review by JoshLove-msft — APPROVED** [review]

File: provision-team/svcbusclient/Program.cs [hunk 5]
```diff
@@ -47,38 +42,30 @@ static void Main(string[] args)
         static async Task MainAsync(string ServiceBusConnectionString, string QueueName, string recipientEmail, string MessageBody)
         {
             Console.WriteLine($"mainAsync");
-            queueClient = new Microsoft.Azure.ServiceBus.QueueClient(ServiceBusConnectionString, QueueName);
+            ServiceBusClient client = new ServiceBusClient(ServiceBusConnectionString);
```
Comment (scottaddie) [inline provision-team/svcbusclient/Program.cs:45]:
```suggestion
            var client = new ServiceBusClient(ServiceBusConnectionString);
```

File: provision-team/svcbusclient/Program.cs [hunk 6]
```diff
@@ -47,38 +42,30 @@ static void Main(string[] args)
...
 
-                    var serializer = DataContractBinarySerializer<Dictionary<string, string>>.Instance;
-                    byte[] data;
-
-                    using (MemoryStream stream = new MemoryStream())
-                    {
-                        serializer.WriteObject(stream, message);
-                        data = stream.ToArray();
-                    }
+                    ServiceBusMessage sendMessage = new ServiceBusMessage(new BinaryData(data));
```
Comment (scottaddie) [inline provision-team/svcbusclient/Program.cs:66]:
```suggestion
                    var sendMessage = new ServiceBusMessage(new BinaryData(data));
```

File: provision-team/svcbusclient/svcbusclient.csproj [hunk 7]
```diff
@@ -1,12 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
```
Comment (scottaddie) [inline provision-team/svcbusclient/svcbusclient.csproj:5]: .NET Core 2.0 is over 3 years out of support. Please update this app to .NET 6.0, which is the latest LTS release. That can be handled in a separate PR, if you prefer.

**Review by scottaddie — APPROVED** [review]

**Review by scottaddie — APPROVED** [review]

**Comment by v-xuto** [comment]
@jongio Please help merge this PR.

**Comment by scottaddie** [comment]
#sign-off

**Pull request merged**